Resources/views/Role/assign_role_by_section.html.twig

```diff
@@ -0,0 +1,51 @@
+{# @var role \eZ\Publish\API\Repository\Values\User\Role #}
+{# @var sections \eZ\Publish\API\Repository\Values\Content\Section[] #}
+{% extends "eZPlatformUIBundle::pjax_admin.html.twig" %}
+
+{% trans_default_domain "role" %}
+
+{% block header_breadcrumbs %}
+    {% set breadcrumb_items = [
+        {link: path('admin_dashboard'), label: 'dashboard.title'|trans({}, 'dashboard')},
+        {link: path('admin_role'), label: 'role.dashboard_title'|trans},
+        {link: path("admin_roleView", {"roleId": role.id}), label: role.identifier},
+        {link: null, label: 'role.assign_role_limit_section'|trans}
+    ] %}
+    {{ parent() }}
+{% endblock %}
+
+{% block header_title %}
+    <h1 class="ez-page-header-name" data-icon="&#xe62d;">
+        {{- 'role.assign_limit_section.user_or_group.pagetitle'|trans({'%roleIdentifier%': role.identifier}) -}}
+    </h1>
+{% endblock %}
+
+{% block content %}
+    <section class="ez-serverside-content">
+        <label for="data-role-assignment-section">{{ 'role.choose_section'|trans }}</label>
+        <select id="data-role-assignment-section" class="data-role-assignment-section">
+            {% for section in sections %}
+            {# @var section \eZ\Publish\API\Repository\Values\Content\Section #}
+            <option value="{{ section.id }}">{{ section.name }}</option>
+            {% endfor %}
+        </select>
+        {# TODO: Section ID 3 is hardcoded below, it should be read from the select above. #}
```

Comment:
I have removed button's attribute as it won't be used. Section ID will be taken directly from select.